README.md

```diff
@@ -1,31 +1,56 @@
 # Solarwinds Orion Integration Pack
 
-This pack intergrates with Solarwinds Orion (a commerical monitoring
+This pack integrates with Solarwinds Orion (a commercial monitoring
 platform).
 
 ## Issues
 
-- If don't have Orion NCM installed extra errors appear in the logs.
+- If you don't have Orion NCM installed extra errors appear in the logs.
 
 ## Configuration
 
+Copy the example configuration in [orion.yaml.example](./orion.yaml.example)
+to `/opt/stackstorm/configs/orion.yaml` and edit as required.
+
+It must contain:
+
 ```yaml
-orion:
```

Comment:
It looks like the config format didn't change at this time but earlier, just readme hasn't been updated, right?